### DISCUSSION TITLE
Getting error when selecting autofill value in prime react input number component

### DISCUSSION THREAD

**Discussion by skyworth01** [primefaces/community#1871] [PrimeReact] [answered]
I am getting an error when i try to select a autofill value 

I am using prime react inputnumber component

![Screenshot 2024-05-10 133213](https://github.com/primefaces/community/assets/75173955/6b9c3723-0749-4daa-a7fd-2cb35968905f)

when i debug ,there is no value in event.Even if to turn off autocomplete ,there is no option for that in input number

How to solve this?

**Comment by melloware** [comment]
Do you mean like browser autofill like this: https://github.com/primefaces/primereact/issues/6345 ?

**Reply by skyworth01** [reply]
Yes.If there was an option to turn off autocomplete,it could be solved.Is there any other way

**Reply by melloware** [reply, marked as answer]
@skyworth01 you can use Passthrough attributes like this...

```xml
<InputNumber  value={value1} onValueChange={(e) => setValue1(e.value)} pt={{ input: { root: { autocomplete: 'off' } } }} />
```

Renders this...

```xml
<span class="p-inputnumber p-component p-inputwrapper p-inputwrapper-filled" data-pc-name="inputnumber" data-pc-section="root">
<input class="p-inputnumber-input p-inputtext p-component p-filled" 
id="integeronly" 
role="spinbutton" 
type="text"
 inputmode="numeric" 
aria-valuenow="42723" 
data-pc-name="inputtext"
 data-pc-section="root" 
value="42,723" 
autocomplete="off">
</span>
```

**Reply by skyworth01** [reply]
@melloware Thank you,Initially I thought about this,but this property was not showing in suggestions,so I thought it won't work.

Thanks again.
